scripts/zones/Bibiki_Bay/mobs/Peerifool.lua

```diff
@@ -0,0 +1,31 @@
+-----------------------------------
+-- Area: Bibiki Bay
+--  mob: Peerifool
+--  Quest: One Good Deed?
+-----------------------------------
+require("scripts/globals/status")
+-----------------------------------
+local entity = {}
+
+entity.onMobInitialize = function(mob)
+    mob:setMobMod(xi.mobMod.ADD_EFFECT, 1)
+end
+
+entity.onAdditionalEffect = function(mob, target, damage)
+    return xi.mob.onAddEffect(mob, target, damage, xi.mob.ae.SLEEP, {chance = 100})
+end
+
+entity.onMobDeath = function(mob, player, isKiller)
+    local party = player:getParty()
+    if #party > 1 then
+        for _, v in ipairs(party) do
+            if player:getZone() == v:getZone() and v:getQuestStatus(xi.quest.log_id.WINDURST, xi.quest.id.windurst.ONE_GOOD_DEED) == QUEST_ACCEPTED then
+                v:setCharVar("Quest[2][92]Prog", 1)
```

Comment:
Would you always want to make this a 1 if accepted. Can someone be on 2, or 3 then help someone and then get downgraded back to 1?